deploy-manage/toc.yml

```diff
@@ -622,6 +622,8 @@ toc:
               - file: users-roles/cluster-or-deployment-auth/authorization-plugins.md
               - file: users-roles/cluster-or-deployment-auth/controlling-access-at-document-field-level.md
               - file: users-roles/cluster-or-deployment-auth/submitting-requests-on-behalf-of-other-users.md
+      - title: "Control access at the document and field level"
```

Comment:
🤔 wonder if the right place for this xref would instead be as a child of [serverless project custom roles](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/3950/deploy-manage/users-roles/serverless-custom-roles) so it essentially sits in two places